<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>901</width>
      <height>341</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="columnCount">
     <number>6</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <column>
     <property name="text">
      <string>Ноги</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Вес</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Домашнее?</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Первая буква вида</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Гендер</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Возраст</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ноги</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Вес</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Домашнее?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Первая буква вида</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Гендер</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>530</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Возраст</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Paws">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>44</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Weight">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>410</y>
      <width>65</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <double>100000000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QRadioButton" name="m">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>490</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>m</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FirstSymbol">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>470</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>1</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="isHouse">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>440</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="w">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>510</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>w</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Age">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>530</y>
      <width>65</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Add">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>88</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
